src/core/p5.Renderer2D.js

```diff
@@ -1383,15 +1383,30 @@ p5.Renderer2D.prototype._applyTextProperties = function() {
 // STRUCTURE
 //////////////////////////////////////////////
 
+// a push() operation is in progress.
+// the renderer should return a 'style' object that it wishes to
+// store on the push stack.
+// derived renderers should call the base class' push() method
+// to fetch the base style object.
```

Comment:
this comment seems like it should go with p5.Renderer rather than p5.Renderer.2D?

Reply:
makes sense. i added it to the base renderer also...